src/components/dialogs/load-modification-dialog.js

```diff
@@ -97,18 +105,55 @@ const LoadModificationDialog = ({
         loading: loadingEquipmentOptions,
     });
 
+    const id = useMemo(() => {
```

Comment:
another time this method that confuses me, is it possible to make it clearer and to avoid duplication ?

Reply:
it will be improved with the refactoring.